src/mahoji/lib/abstracted_commands/maniacalMonkeyCommand.ts

```diff
@@ -0,0 +1,315 @@
+import { reduceNumByPercent, Time } from 'e';
+import { Bank } from 'oldschooljs';
+
+import { getSimilarItems } from '../../../lib/data/similarItems';
+import { trackLoot } from '../../../lib/lootTrack';
+import { resolveAttackStyles } from '../../../lib/minions/functions';
+import { SkillsEnum } from '../../../lib/skilling/types';
+import { Skills } from '../../../lib/types';
+import { formatDuration, formatMissingItems, hasSkillReqs, itemID } from '../../../lib/util';
+import addSubTaskToActivityTask from '../../../lib/util/addSubTaskToActivityTask';
+import { calcMaxTripLength } from '../../../lib/util/calcMaxTripLength';
+import resolveItems from '../../../lib/util/resolveItems';
+import { updateBankSetting } from '../../../lib/util/updateBankSetting';
+import { maniacalMonkeyID } from '../../../tasks/minions/maniacalMonkeyActivity';
+import { PvMMethod } from './../../../lib/constants';
+import { iceBarrageConsumables, iceBurstConsumables } from './../../../lib/minions/data/combatConstants';
+import { Consumable } from './../../../lib/minions/types';
+import { ManiacalMonkeyTaskOptions } from './../../../lib/types/minions';
+import { applySkillBoost } from './minionKill';
+
+export async function maniacalMonkeyCommand(
+	user: MUser,
+	channelID: string,
+	quantity: number | undefined,
+	method: PvMMethod | undefined
+) {
+	const { minionName } = user;
+
+	const skillReqs: Skills = {
+		hitpoints: 74,
+		prayer: 74
+	};
+
+	if (!method) {
+		method = 'none';
+	}
+
+	const [hasReqs] = hasSkillReqs(user, skillReqs);
+	if (!hasReqs) {
+		return `You not meet skill requirements, you need ${Object.entries(skillReqs)
+			.map(([name, lvl]) => `${lvl} ${name}`)
+			.join(', ')}.`;
+	}
+	const magicLevel = user.skillLevel('magic');
+	const rangedLevel = user.skillLevel('ranged');
+	if ((method === 'burst' || method === 'barrage') && magicLevel < 62) {
```

Comment:
Later in the code you compare magic level to be minimum 70 for training magic (bursting) so it may be better to enforce level 70 here and remove that check later down the line?